core/frontend-devtools/README.md

```diff
@@ -188,23 +188,27 @@ Reality models can be attached to a display style to provide context when that d
 ### Planar Mask keyins
 
 This package supplies several keyins to control planar masking for background maps and reality models. These masks provide a two and a half dimensional method for masking the regions where the background map, reality models and BIM geometry overlap.
-Planar masks include an optional transparency override.  If a transparency values is specified, a value of 0 will completely mask the reality model  - the masked portion will be omitted completely, higher values will produce less masking and a semit translucent display of the masked geometry. If no transparency is included then the transparency value from the mask elements is used.
+Planar masks include an optional transparency override.  If a transparency values is specified, a value of 0 will completely mask the reality model  - the masked portion will be omitted completely, higher values will produce less masking and a semit translucent display of the masked geometry. If no transparency is included then the transparency value from the mask elements is used.  An additional argument boolean argument will invert the mask and mask the area outside the mask geometry rather than inside.
 
 #### Background map planar masks
 
 These keyins control the planar masking of the background map.
 
 * `fdt set map mask by priority`  Set the background map to be masked based on priority.  Masking by priority will mask by higher priority models.  By default background map have lowest priorty (-2048) Unless their priority is overridden reality models are higher priority and BIM models are always higher priority, therefore if priority masking is selected then the background map will be masked by all reality and BIM models. A value for transparency may optionally be included.
   * `transparency`: (optional) - A value for the mask transparency override. [0...1]
-  * `priority`: (optional) - A bias to be applied to the default reality model priority.  As lower priority models are masked by higher priority ones, specifying a priority of -1 would cause the reality model to be masked by other reality models.
+  * `invert`: (optional) - If `1 ` or `true` then the mask sense is inverted and the area outside the mask geometry will be masked and the area inside the mask is displayed.
```

Comment:
"1" does not work. It needs to be one of the following (case-insensitive):
```
export function parseBoolean(arg: string | undefined): boolean | undefined {
  if (undefined === arg)
    return undefined;

  switch (arg.toLowerCase()) {
    case "on": return true;
    case "true": return true;
    case "off": return false;
    case "false": return false;
    default: return undefined;
  }
```

Reply:
Fixed.